### DISCUSSION TITLE
Cicero: seqlengths is NULL

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#863] [Q&A] [answered]
Hi Tim,
I am trying to find co-accessible networks of my scATAC data by following this pipeline https://satijalab.org/signac/articles/cicero.html
![Screen Shot 2021-11-11 at 9 07 30 AM](https://user-images.githubusercontent.com/54915282/141321041-99b747b8-cd61-4e99-902b-773baf1132c2.png)
However, the seqlengths of my Seurat object is NULL probably since the species I am working on is not human or mouse.

Is there anyway that I can  construct this co-accessible network using Cicero? Can you give me some suggestions?

Thanks
Best
Penny

**Comment by timoast** [comment, marked as answer]
The `seqlengths()` function is just a way to access the length of each chromosome in the dataset. If you don't have the `seqinfo` information set for your object, you can construct this in other ways. I'd suggest looking at the `genome.df` object shown in the vignette, and then create the equivalent dataframe for your species
